examples/text_classification_with_scriptable_tokenzier/handler.py

```diff
@@ -0,0 +1,107 @@
+"""
+Module for text classification with scriptable tokenizer
+DOES NOT SUPPORT BATCH!
+"""
+from abc import ABC
+import logging
+
+import torch
+import torch.nn.functional as F
+# Necessary to successfully load the model (see https://github.com/pytorch/text/issues/1793)
+import torchtext  # pylint: disable=unused-import
+
+from ts.torch_handler.base_handler import BaseHandler
+from ts.utils.util import map_class_to_label, CLEANUP_REGEX
+
+
+logger = logging.getLogger(__name__)
+
+def remove_html_tags(text):
+    """
+    Removes html tags
+    """
+    clean_text = CLEANUP_REGEX.sub("", text)
+    return clean_text
+
+class TextClassifier(BaseHandler, ABC):
+    """
+    TextClassifier handler class. This handler takes a text (string) and
+    as input and returns the classification text based on the model vocabulary.
+    """
+
+    def preprocess(self, data):
+        """
+        Tokenization is dealt with inside the scripted model itself.
+        We therefore only apply these basic cleanup operations :
+            - remove html tags
+            - lowercase all text
+
+        Args:
+            data (str): The input data is in the form of a string
+
+        Returns:
+            (Tensor): Text Tensor is returned after perfoming the pre-processing operations
+            (str): The raw input is also returned in this function
+        """
+
+        # Compat layer: normally the envelope should just return the data
+        # directly, but older versions of Torchserve didn't have envelope.
+        # Processing only the first input, not handling batch inference
+
+        line = data[0]
+        text = line.get("data") or line.get("body")
+        if isinstance(text, (bytes, bytearray)):
+            text = text.decode('utf-8')
+
+        text = remove_html_tags(text)
+        text = text.lower()
+
+        return text
+
+    def inference(self, data, *args, **kwargs):
```

Comment:
looks like optional args are not used?

Reply:
These come from the BaseHandler. Removing them results in a linter error as the overriding method has a different signature. Any advice? Could not find a compelling solution other than disable one of the linter errors (unused variable or signature change).